plugins/modules/zos_stat.py

```diff
@@ -2275,6 +2361,17 @@ def get_data_set_handler(
     except (GDSNameResolveError, Exception):
         return DataSetHandler(name, exists=False)
 
+    alias_name = None
+
+    try:
+        is_an_alias, base_name = get_name_if_data_set_is_alias(name, module, tmp_hlq=tmp_hlq)
+    except:
```

Comment:
This line kicks out a warning and error for 'bare except'.  (no condition or parameter about which exception(s) are covered.

Reply:
Done!